src/main/java/com/amazon/opendistro/elasticsearch/performanceanalyzer/PerformanceAnalyzerApp.java

```diff
@@ -161,6 +161,7 @@ private static void startRcaController(ClientServers clientServers) {
             RcaConsts.RCA_CONF_IDLE_MASTER_PATH,
             RcaConsts.RCA_CONF_PATH,
             RcaConsts.networkPollerPeriodicity,
```

Comment:
We can use this opportunity to have a const in RcaCOnst for each poller and name the `networkPollerPeriodicityTimeUnit` as `rcaPollerPeriodicityTimeUnit` ? what do you think ?

Reply:
Yeah , we can do that. I just kept same constant for other pollers as they all had same periodicity and if we wanted to change periodicity in future then it will a couple more lines of code change.